src/notification.cpp

```diff
@@ -257,24 +257,75 @@ class NotificationPrivate : public NotificationData
 };
 
 /*!
+    \qmlmodule org.nemomobile.notifications
```

Comment:
It also causes the generation to contain the 'import org.nemomobile.notifications . ' line, which is what we want (well, it would be nice to have the version numbers, too).